plugins/notifications-backend-module-email/src/alpha.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import type { BackendDynamicPluginInstaller } from '@backstage/backend-dynamic-feature-service';
+import { notificationsModuleEmail } from './module';
+
+/** @alpha */
+export const dynamicPluginInstaller: BackendDynamicPluginInstaller = {
```

Comment:
No problem having this in place for now, but eventually this shouldn't be needed except for very specific circumstances.